mbtiles/scripts/cli.py

```diff
@@ -72,20 +73,25 @@ def validate_nodata(dst_nodata, src_nodata, meta_nodata):
 @click.option('--resampling', type=click.Choice(RESAMPLING_METHODS),
               default='nearest', show_default=True,
               help="Resampling method to use.")
+@click.option('--bands', type=int,
+              default=None, show_default=True,
+              help="Number of bands in the output dataset (4 for RGBA)")
+@click.option('--colormap',
+              default=None, show_default=True,
+              help='''Colormap for a 1-band PNG file as a JSON array (e.g. {"0":[1,2,3,4],"23":[9,4,3,2]} for RGBA values for pixel values 0 and 23)''')
```

Comment:
Would it be more accurate to say that the colormap is a JSON object where the keys are integers and the values are arrays of integers?

A colormap-file option would be nice, too, maybe in another PR after this one is merged.